javascript/javascript2/week1/danish_exercise/danish.js

```diff
@@ -0,0 +1,19 @@
+const danishString = "Jeg har eøæn blå bil";
+function getDanish(str) {
+ let m = str.match(/[æøå]/gi);
+   const count = (m === null) ? 0 : m.length;
+    let a =str.match(/[æ]/gi);
+    const count1 = (a === null) ? 0 : a.length;
+   let b = str.match(/[ø]/gi);
+   const count2 = (b === null) ? 0 : b.length;
+   let c = str.match(/[å]/gi);
```

Comment:
Instead of checking each and every danish letters, you can put them inside an array and then use the loops or array methods to check the condition and don't use a, b, etc for variables

Reply:
Sure,I shall try it.